imagersite/imager_profile/views.py

```diff
@@ -1,3 +1,167 @@
-from django.shortcuts import render
+from django.views.generic import TemplateView, ListView, DetailView, FormView
+from imager_images.models import Photos, Album, Face
+from .models import ImagerProfile
+from .forms import ProfileSettingsForm
+from django.http import HttpResponse
 
-# Create your views here.
+
+def get_faces(photo):
```

Comment:
This seems like an odd place for this to be.  Isn't it more related to the images?